lib/adapters/REST/endpoints/concept-scheme.ts

```diff
@@ -77,3 +109,23 @@ export const update: RestEndpoint<'ConceptScheme', 'update'> = (
     }
   )
 }
+
+export const updatePut: RestEndpoint<'ConceptScheme', 'updatePut'> = (
+  http: AxiosInstance,
+  params: UpdateConceptSchemeParams,
+  data: CreateConceptSchemeProps,
+  headers?: RawAxiosRequestHeaders
+) => {
+  return raw.put<ConceptSchemeProps>(
+    http,
+    `${basePath(params.organizationId)}/${params.conceptSchemeId}`,
+    data,
+    {
+      headers: {
+        'X-Contentful-Version': params.version,
+        'Content-Type': 'application/json-patch+json',
```

Comment:
wondering if this header is intended here in `updatePut` since it is normally used only for PATCH reqs

Reply:
Oh good catch! Removed.